### ISSUE TITLE
ext-mbstring should be required, not suggested

### ISSUE THREAD

**Issue by oschwald** [Respect/Validation#646]
Currently `ext-mbstring` is listed as a suggested dependency, but `Respect\Validation\Rules\Length` will throw an exception if `mb_strlen` is missing:

```
PHP Fatal error:  Call to undefined function Respect\Validation\Rules\mb_strlen() in vendor/respect/validation/library/Rules/Length.php on line 57
```

Alternatively, as mentioned in #319, it would be relatively simple to conditionally use `mb_strlen`.

**Comment by johnnymn** [comment]
+1

**Comment by henriquemoody** [comment]
Those bugs were finally fixed on versions [1.0.12](https://packagist.org/packages/respect/validation#1.0.12) and [1.1.7](https://packagist.org/packages/respect/validation#1.1.7) by #729.

Thanks for reporting! :panda_face:
